tests/common/tensor_test.py

```diff
@@ -238,7 +238,7 @@ def test_last_dim_softmax_does_softmax_on_last_dim(self):
 
     def test_last_dim_softmax_handles_mask_correctly(self):
         batch_size = 1
-        length_1 = 5
```

Comment:
I.e., this change made the test fail when I first made it.  It's not an issue anymore, because I fixed it in that other PR, but still probably better to have this changed.